contexts/deposit-context.tsx

```diff
@@ -141,7 +141,7 @@ export const DepositProvider = ({ children }: React.PropsWithChildren<{}>) => {
           functionName: 'deposit',
           args: [
             asset.substitutes[0].address,
-            amount,
+            amount + minimumInterestEarned,
             epochs,
             minimumInterestEarned,
             { deadline, v, r, s },
```

Comment:
@onemonster Could you write todo for `PermitAmount`?

Reply:
I think the issue I wrote will suffice